src/basic/terminal-util.c

```diff
@@ -11,6 +11,7 @@
 #include <stdarg.h>
 #include <stddef.h>
 #include <stdlib.h>
+#include <sys/file.h>
```

Comment:
Then, this is not necessary anymore.

Reply:
I don't see where this file would have a `lock-util.h` inclusion.